app/assets/stylesheets/styles/components/pages/_home.css.scss

```diff
@@ -115,10 +116,16 @@ video { display: block; }
         }
       }
     }
+    .crops-loading-spinner {
+      position: absolute;
+      background: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.